src/gatk_gcnv_caller.sh

```
@@ -0,0 +1,655 @@
+
+    SECONDS=0
+    echo "Uploading ploidy and basecounts for sub jobs"
+
+    total_files=$(find /home/dnanexus/inputs/ploidy_dir /home/dnanexus/inputs/basecounts -type f | wc -l)
+    find /home/dnanexus/inputs/ploidy_dir /home/dnanexus/inputs/basecounts -type f \
+        | xargs -P $PROCESSES -n1 -I{} bash -c "_upload_single_file {} _ false"
+
+    duration=$SECONDS
+    echo "Uploaded ${total_files} ploidy and base count files in $(($duration / 60))m$(($duration % 60))s"
+
+    for interval in $(find /home/dnanexus/inputs/scatter-dir -type f -name "scattered.interval_list"); do
```

Shellcheck

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)